cmd/login.go

```diff
@@ -0,0 +1,188 @@
+package cmd
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/user"
+	"strings"
+	"syscall"
+
+	"golang.org/x/crypto/ssh/terminal"
+
+	"github.com/spf13/cobra"
+	"github.com/turnerlabs/harbor-auth-client"
+)
+
+//Auth struc
+type Auth struct {
+	Version  string `json:"version"`
+	Username string `json:"username"`
+	Token    string `json:"token"`
+}
+
+// loginCmd represents the login command
+var loginCmd = &cobra.Command{
+	Use:   "login",
+	Short: "Login to harbor",
+	Long:  ``,
+	Run:   login,
+}
+
+var authURL = "https://auth.services.dmtio.net"
+
+func init() {
+	RootCmd.AddCommand(loginCmd)
+}
+
+func login(cmd *cobra.Command, args []string) {
+	_, _, err := Login()
+	if err != nil {
+		//log.Printf(err.Error())
+		fmt.Println("Login Failed")
+	}
+	return
+}
+
+func writeFile(version string, username string, token string) (bool, error) {
+	usr, err := user.Current()
+	if err != nil {
+		return false, err
+	}
+
+	var path = usr.HomeDir + "/.harbor"
+	err = os.Chdir(path)
```

Comment:
I got the following running it the first time.  Is there a way to suppress this?

```
2016/08/16 15:55:34 chdir /Users/jritsema/.harbor: no such file or directory
```